src/components/Select/Select.test.tsx

```diff
@@ -15,45 +14,46 @@ describe("Select", () => {
           { value: "2", label: "Bionic Beaver" },
           { value: "3", label: "Xenial Xerus" },
         ]}
+        wrapperClassName="select"
       />
     );
-    expect(wrapper).toMatchSnapshot();
+    // eslint-disable-next-line testing-library/no-node-access
+    expect(document.querySelector("select")).toMatchSnapshot();
```

Comment:
```suggestion
    expect(screen.getByRole("combobox")).toMatchSnapshot();
```

Reply:
Hah, no idea why I did it like that! All the other tests in the file use `screen.getByRole("combobox")`. I've fixed it.